app.js

```diff
@@ -30,12 +31,47 @@
       this.$('.query.date').datepicker({ dateFormat: "yy-mm-dd" });
     }),
 
+    // Toggles acending/decending order of the column header clicked
+    sortTable: function(event) {
+      var $tr = this.$(event.target);
+      var position = $tr.index();
+      var $tableBody = $tr.closest('table').find('tbody');
+      var greaterThan,
+          lessThan;
+
+      $tr.addClass('sorted');
+      $tr.siblings().removeClass('sorted ascending');
+      $tr.toggleClass('ascending');
+
+      if ( $tr.hasClass('ascending') ) {
+        greaterThan = 1;
+        lessThan = -1;
+      } else {
+        greaterThan = -1;
+        lessThan = 1;
+      }
+
+      var newList = $tableBody.find('tr').sort( function(a,b) {
+        if ( this.$(a).find('td:eq(' + position + ')').text() > this.$(b).find('td:eq(' + position + ')').text() ){
```

Comment:
Untested, but should be something similar to the following, according to http://www.w3schools.com/jsref/jsref_sort.asp:
```
var newList = $tableBody.find('tr').sort( function(a,b) {
  if ($tr.hasClass('ascending')){
    return this.$(a).find('td:eq(' + position + ')').text() - this.$(b).find('td:eq(' + position + ')').text()
  } else {
    return this.$(b).find('td:eq(' + position + ')').text() - this.$(a).find('td:eq(' + position + ')').text()
  }
}.bind(this));
```

This way you don't need the vars greaterThan and lessThan.